### PR TITLE
feat(resource): Import role collection for globalaccount // Deactivate role

### PR DESCRIPTION
**Pull request by lechnerc77**
## Purpose

This PR:

* Deactivates the resource `globalaccount_role`
* adds the import functionality 
* adds information about import to documentation

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
Import resources according to documentation
```

## What to Check

Verify that the following are valid

* `tfstate` file is imported as expected

## Other Information

Deactivation of role resource based on usage pattern of BTP CLI: CUD operations do not seem to be used

### CODE REVIEW

**Comment by lechnerc77** [comment]
Sonar Cloud check fails, test for roles currently neglected.

File: internal/provider/resource_globalaccount_role_collection_test.go [hunk 1]
```diff
@@ -51,6 +80,6 @@ func hclResourceGlobalAccountRoleCollection(resourceName string, displayName str
 	return fmt.Sprintf(`resource "btp_globalaccount_role_collection" "%s" {
         name         = "%s"
         description  = "%s"
-	roles        = %v
+	    roles        = %v
```
Comment (v0lkc) [inline internal/provider/resource_globalaccount_role_collection_test.go]: Could you please run `go fmt ./...` again? This doesn't look properly formatted to me.

**Review by v0lkc — CHANGES_REQUESTED** [review]

File: internal/provider/resource_globalaccount_role_collection_test.go (hunk 1 above)
Comment (lechnerc77) [reply internal/provider/resource_globalaccount_role_collection_test.go]: Fixed, it was tab vs. spaces

Comment (lechnerc77) [reply internal/provider/resource_globalaccount_role_collection_test.go]: go fmt does not resolve that however

**Review by rui1610 — APPROVED** [review]
lgtm

**Pull request merged**